.github/workflows/lint.yml

```diff
@@ -1,26 +0,0 @@
-name: Lint code
-on:
-  push:
-  pull_request:
-
-jobs:
-  PythonBlack:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Check code lints with Black
-        uses: psf/black@stable
-
-  Prettier:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: actions/setup-node@v2
-
-      - name: Install Prettier
-        run: npm install -g prettier
-
-      - name: Run Prettier --check
-        run: prettier --check ${GITHUB_WORKSPACE}
```

Comment:
If possible it would be nice to keep Prettier in place, for linting the markdown + YAML files.. (maybe you're already on the case for this..)

Reply:
Ahhh yeah I was wondering why this was here.
My mistake, adding it back :)

Reply:
I'll see if I can embed it into pre-commit ;)

Reply:
No worries! Should be possible I think: https://prettier.io/docs/en/precommit.html

Reply:
Done :v: Let me know what you think :)